SConstruct

```diff
@@ -349,6 +349,10 @@ def scons(): # pylint: disable=too-many-locals
 
     platform_arm = is_platform_arm()
 
+    if 'VIRTUAL_ENV' in os.environ:
+        env.PrependENVPath('PATH', f"{os.environ['VIRTUAL_ENV']}/bin")
+        env['ENV']['VIRTUAL_ENV'] = os.environ['VIRTUAL_ENV']
```

Comment:
(pylint-invalid-sequence-index) Sequence index is not an int, slice, or instance with __index__